src/tests/ampDefaultEpic.ts

```diff
@@ -0,0 +1,15 @@
+import { getLocalCurrencySymbol } from '../lib/geolocation';
+
+export const ampDefaultEpic: object = (geolocation?: string) => {
```

Comment:
Is it worth typing this as a `Test` rather than `object`? I'm not sure, as it wouldn't have an immediate practical benefit, though I do like keeping the data structures in the app to a minimum.

Reply:
I'm not at all wedded to this, it was just to appease the linter, as the function declaration didn't previously have an explicit return type. The function now actually stringifies the object as JSON, and returns it as a plain string.

Reply:
I think there'd be value in defining a new `AMPEpic` type, but it's not a `Test` at this point - I don't think it'd be possible to have the client decide the variant